feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/domain/nft/list/NftListInteractor.kt

```diff
@@ -44,4 +59,27 @@ class NftListInteractor(
     suspend fun fullSyncNft(nft: Nft) {
         nftRepository.fullNftSync(nft)
     }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    fun subscribeNftOwnerChanged(): Flow<Unit> {
+        return nftRepository.getPendingSendTransactionsNftIds()
+            .flatMapLatest(::subscribeNftsOwnerAddresses)
+            .onEach {
+                val blockTimeInMillis = chainStateRepository.expectedBlockTimeInMillis(it.chainId)
+                delay(blockTimeInMillis.toLong())
+                syncNftsList()
```

Comment:
I left `syncNftsList()` because I need to change new actual state of blockchain state but I change only necessary data in DB. I optimized it a little bit and removed `delay`